.github/workflows/mi210.yml

```diff
@@ -96,7 +96,90 @@ jobs:
-        run: make -j16
+        run: make -j12 all
+
+      - name: test
+        working-directory: kokkos-kernels/build
+        run: ctest --output-on-failure -V --timeout 3600
+
+  PR_VEGA908_ROCM561_HIP_SERIAL_LEFT_OPENBLAS_OPENLAPACK_REL:
+    name: PR_VEGA908_ROCM561_HIP_SERIAL_LEFT_OPENBLAS_OPENLAPACK_REL
+    runs-on: [kk-env-openblas-0.3.23-hip-5.6.1-latest]
+    
+    steps:
+      - name: checkout_kokkos_kernels
+        uses: actions/checkout@v4
+        with:
+          path: kokkos-kernels
+
+      - name: checkout_kokkos
+        uses: actions/checkout@v4
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ github.base_ref }}
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          mkdir -p kokkos/{build,install}
+          cd kokkos/build
+          HIPCC=$(which hipcc)
+          cmake -DCMAKE_CXX_COMPILER=$HIPCC \
+            -DCMAKE_CXX_FLAGS=-O3 \
+            -DCMAKE_EXE_LINKER_FLAGS= \
+            -DCMAKE_INSTALL_PREFIX=$PWD/../install \
+            -DKokkos_ENABLE_SERIAL=ON \
+            -DKokkos_ENABLE_HIP=ON \
+            -DKokkos_ARCH_VEGA90A=ON \
+            -DKokkos_ENABLE_TESTS=OFF \
+            -DKokkos_ENABLE_EXAMPLES=OFF \
+            -DCMAKE_VERBOSE_MAKEFILE=ON \
+            -DCMAKE_CXX_EXTENSIONS=OFF \
+            -DCMAKE_CXX_STANDARD=17 \
+            -DBUILD_SHARED_LIBS=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            ..
+
+      - name: build_and_install_kokkos
+        working-directory: kokkos/build
+        run: make -j16 install
+
+      - name: configure_kokkos_kernels
+        run: |
+          mkdir -p kokkos-kernels/{build,install}
+          cd kokkos-kernels/build
+          HIPCC=$(which hipcc)
+          cmake -DCMAKE_CXX_COMPILER=$HIPCC \
+            -DKokkos_DIR=$PWD/../../kokkos/install/lib64/cmake/Kokkos \
+            -DCMAKE_CXX_FLAGS="-O3 " \
+            -DCMAKE_INSTALL_PREFIX= \
+            -DKokkosKernels_ENABLE_TESTS_AND_PERFSUITE=OFF \
+            -DKokkosKernels_ENABLE_TESTS=ON \
+            -DKokkosKernels_ENABLE_PERFTESTS=ON \
+            -DKokkosKernels_ENABLE_EXAMPLES:BOOL=ON \
+            -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=OFF \
+            -DKokkosKernels_INST_COMPLEX_DOUBLE=ON \
+            -DKokkosKernels_INST_DOUBLE=ON \
+            -DKokkosKernels_INST_ORDINAL_INT=ON \
+            -DKokkosKernels_INST_OFFSET_SIZE_T=ON \
+            -DKokkosKernels_INST_OFFSET_INT=ON \
+            -DKokkosKernels_INST_LAYOUTLEFT=ON \
+            -DKokkosKernels_ENABLE_TPL_CUSOLVER=OFF \
+            -DKokkosKernels_ENABLE_TPL_CUSPARSE=OFF \
+            -DKokkosKernels_ENABLE_TPL_ROCSOLVER=ON \
+            -DKokkosKernels_ENABLE_TPL_ROCSPARSE=ON \
+            -DKokkosKernels_ENABLE_TPL_BLAS=ON \
+            -DKokkosKernels_ENABLE_TPL_CUBLAS=OFF \
```

Comment:
You can also remove cublas

Reply:
Same as above. I will triple-check but I will leave these to match the old configs from cm_test_all_sandia. All of these are global state and can have all kinds of 'fun' side affects.

Reply:
Dropping the disabled CUSOLVER and/or CUSPARSE options in non-Cuda builds should be safe to test, that is there as an artifact of some clunkiness around the parsing and setting of TPL options here https://github.com/kokkos/kokkos-kernels/blob/4f3220c911c0a3ef15fe22c7e2949b9f5f0ee720/cm_generate_makefile.bash#L174-L240